web/src/app/globals.css

```
@@ -174,6 +174,12 @@ html {
   padding: 1rem;
   overflow-x: auto;
   margin: 1rem 0;
+  /* Code blocks here are as often a drafted message/paragraph (no real line
+     breaks) as actual code — unwrapped + scrolling hides most of a one-line
+     194-char draft behind a scrollbar instead of just letting it read. Real
+     code with meaningful indentation still reads fine wrapped. */
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)